src/Http/Http.Abstractions/src/Validation/ValidatableTypeInfo.cs

```diff
@@ -44,12 +43,15 @@ protected ValidatableTypeInfo(
     /// <inheritdoc />
     public virtual async Task ValidateAsync(object? value, ValidateContext context, CancellationToken cancellationToken)
     {
-        Debug.Assert(context.ValidationContext is not null);
         if (value == null)
         {
             return;
         }
 
+        // Although classes can be annotated with [DisplayName], we only process display names when producing
```

Comment:
Seems fine. Future looking, any reason we might prefer `[DisplayName]` here?

Reply:
I think the biggest issue would be that the default behavior here is inconsistent with how we handle properties and parameters where we do respect the attribute.

Also, this comment should be updated to say `[Display(Name = "")]`. 😓